drivers/afe/ad4110/ad4110.h

```diff
@@ -204,29 +229,47 @@ enum ad4110_gain {
 struct ad4110_dev {
 	/* SPI */
 	struct spi_desc			*spi_dev;
-	/* GPIO */
-	struct gpio_desc		*gpio_reset;
 	/* Device Settings */
+	enum ad4110_voltage_reference volt_ref;
 	enum ad4110_state		data_stat;
 	enum ad4110_data_word_length 	data_length;
 	enum ad4110_adc_crc_mode 	adc_crc_en;
 	enum ad4110_afe_crc_mode	afe_crc_en;
 	enum ad4110_op_mode 		op_mode;
 	enum ad4110_gain 		gain;
+	enum ad4110_sync_en sync;
+	enum ad4110_afe_clk_cfg afe_clk;
+	enum ad4110_adc_clk_sel adc_clk;
+	uint8_t				addr;
+	/* GPIO - used only for continuous mode */
+	struct irq_ctrl_desc *irq_desc;
+	int32_t nready_pin;
```

Comment:
use `uint32_t`, since you pass this member to:

https://github.com/analogdevicesinc/no-OS/blob/85f3a3a1baf878fe30345c24943ac49b435b519a/include/irq.h#L183

https://github.com/analogdevicesinc/no-OS/blob/85f3a3a1baf878fe30345c24943ac49b435b519a/include/irq.h#L186